include/swift/SIL/SILBasicBlock.h

```diff
@@ -412,6 +412,10 @@ public llvm::ilist_node<SILBasicBlock>, public SILAllocated<SILBasicBlock> {
 
   void eraseInstructions();
 
+  // SWIFT_ENABLE_TENSORFLOW
+  /// Get estimated code size, for debugging only.
+  unsigned codeSize() const { return InstList.size(); }
```

Comment:
nit: Make the name more direct? say `numInstructions`

Reply:
I'm thinking that we may want to tweak the "code size" metric definition in the future (say add weightage based on inst types), so having a more abstract API name could help.